_inc/client/state/connection/actions.js

```diff
@@ -309,11 +315,11 @@ export const reconnectSite = ( action = 'reconnect' ) => {
 				dispatch(
 					createNotice(
 						'is-error',
-						__( 'There was an error reconnecting Jetpack. Error: %(error)s', {
-							args: {
-								error: error,
-							},
-						} ),
+						sprintf(
+							/* translators: placeholder is the error. */
+							__( 'There was an error reconnecting Jetpack. Error: %s', 'jetpack' ),
+							error.response.message || error.response.code
```

Comment:
@sergeymitr @jeherve fyi this was accidentally [reverted in a previous commit ](https://github.com/Automattic/jetpack/pull/16475/commits/bbd297688a0ba4eb44e692ae71dd9bacfb0662d8)and took the opportunity to fix it here

Reply:
Good catch, thanks 👍